design/vela-core/componenttrait-composing-and-trait-CR-naming.md

```diff
@@ -3,7 +3,7 @@
 * Owner: Jianbo Sun (@wonderflow), Zhou Zheng Xi (@zzxwill)
 * Reviewers: KubeVela/Crossplane Maintainers
 * Status: Draft
-
+* Notice: Manual scaler is deprecated. See this [issue](https://github.com/oam-dev/kubevela/issues/2262) to get more information.
```

Comment:
Great job! Can you create or find a doc introducing the replacement in kubevela.io and add the link here?

Reply:
ok,I finish this as soon as possible